Sports.py

```diff
@@ -2,49 +2,248 @@
+        self.assists_color = graphics.Color(0, 255, 255) # Cyan 
+        self.rebounds_color = graphics.Color(0, 0, 255) # Blue 
+        self.blocks_color = graphics.Color(128, 0, 128) # Purple 
+        self.steals_color = graphics.Color(255, 0, 0) # Red 
+        self.points_color = graphics.Color(255, 255, 0) # Yellow 
+        self.field_goals_percent_color = graphics.Color(0, 128, 128) # Teal 
+        self.three_pointers_percent_color = graphics.Color(255, 165, 0) # Orange 
+        self.free_throws_percent_color = graphics.Color(173, 216, 230) # Light Blue 
+        
+        self.error_color = graphics.Color(255, 0, 0) # Red 
+        self.no_stats_color = graphics.Color(255, 255, 0) # Yellow
+        self.updating_stats_color =graphics.Color(0, 255, 0) #Green
+
+
+        self.datetime_color = graphics.Color(255, 255, 255) # WHITE 
+        self.teamstats = graphics.Color(0, 255, 255)# CYAN -- still working on it.
+        
+    def display_image(self, offscreen_canvas, image_path):
+        # Open the image using PIL
+        image = Image.open(image_path)
+        # Resize the image to fit display
+        image = image.resize((128, 58), self.resample_filter)
+        
+        # Convert the image to RGB format
+        image = image.convert('RGB')
+        
+        # Display the image on the LED matrix
+        for x in range(128):
+            for y in range(58):
+                r, g, b = image.getpixel((x, y))
+                offscreen_canvas.SetPixel(x, y, r, g, b)
+
+                
+    def loadPlayersFromFile(self):
+        self.players = []
+        try:
+            with open('/home/timkosinski/rpi-rgb-led-matrix/bindings/python/samples/Players.txt','r') as file:
+                line = file.readline()
+                player_names = line.strip().split(',')
+                for name in player_names:
+                   self.players.append(PlayerStats(name, "random", "0", "0", "0", "0", "0", "0%", "0%", "0%"))
+   
+        except IOError as e:
+            print(f"An error occurred while reding the file: {e}")
+            return
+                
+    def populatePlayerStats(self):
+        api = GameFacade()
+        text_height = 6
+        x_pos = 2
+        vertical_pos = 64
+      
+     
+        
+        # Prepare the offscreen canvas
+        offscreen_canvas = self.matrix.CreateFrameCanvas()
+
+        # Clear the screen
+        self.clearScreen(offscreen_canvas)
+
+        # Display the image for a certain period
+        self.display_image(offscreen_canvas, '/home/timkosinski/rpi-rgb-led-matrix/bindings/python/samples/Logo3.png')
+        offscreen_canvas = self.matrix.SwapOnVSync(offscreen_canvas)
+       
+
+       
+        for player in self.players:
+            time.sleep(1)
+            
+            try:                                                        #2024,APR,07
+                player_stats_list = api.get_player_stats(player.name, "2024", "04", "07")
+                time.sleep(1)

```

Comment:
Is this for a certain day? was this for test or not live stats?

Reply:
I had helped him last night set them back to live stats.